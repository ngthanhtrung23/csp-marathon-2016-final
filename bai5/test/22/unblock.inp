.12223
41..53
41005.
667.5.
..7888
.99AA.
