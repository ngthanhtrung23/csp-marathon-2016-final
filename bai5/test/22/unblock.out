27
3 D 1
4 U 1
9 L 1
7 D 1
6 R 2
1 D 1
2 L 1
5 U 1
6 R 1
7 U 1
A R 1
9 R 2
1 D 2
0 L 2
7 U 2
6 L 1
5 D 1
2 R 2
7 U 1
0 R 2
1 U 2
8 L 2
3 D 2
9 L 2
A L 2
5 D 2
0 R 2

